### PR TITLE
Yaml issue templates

### PR DESCRIPTION
**Pull request by alex-merkert-olive**
### Changes made:

- Added yaml configurations for issue templates instead of using markdown

### CODE REVIEW

**Review by racary-olive — APPROVED** [review]

**Review by harrisonkunkel — APPROVED** [review]

**Review by jeffsun-olive — APPROVED** [review]

**Comment by iangallOlive** [comment]
**Looks** good, but did you have any way to preview/test this?

**Comment by alex-merkert-olive** [comment]
> **Looks** good, but did you have any way to preview/test this?

I couldn't really _test_ it, but I did at least get to see previews here:
https://github.com/open-olive/loop-development-kit/blob/yaml-issue-templates/.github/ISSUE_TEMPLATE/bug_report.yaml
https://github.com/open-olive/loop-development-kit/blob/yaml-issue-templates/.github/ISSUE_TEMPLATE/feature_request.yaml

**Review by MaggieMaSS — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Bug Report
```
Comment (iangallOlive) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:1]: Wrong name!

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+description: Create a report to help us improve
```
Comment (iangallOlive) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:2]: Maybe this should change too?

Comment (alex-merkert-olive) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: done

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 1 above)
Comment (alex-merkert-olive) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: thanks!

**Pull request merged**